src/Microsoft.Extensions.CodeGeneration.Utils/DotNet/ILibraryManager.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.DotNet.ProjectModel;
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.Extensions.CodeGeneration.DotNet
+{
+    public interface ILibraryManager
```

Comment:
This was part of it, but for the DNX world. 
As communicated [here](https://github.com/aspnet/Announcements/issues/149) it needs to be replaced.